docs/platforms/javascript/guides/angular/features/component-tracking.mdx

```diff
@@ -122,58 +74,44 @@ import * as Sentry from "@sentry/angular";
   templateUrl: "./login.component.html",
 })
 export class LoginComponent implements OnChanges {
-  @Sentry.TraceMethodDecorator()
-  ngOnChanges(changes: SimpleChanges) {}
+  @Sentry.TraceMethod({ name: "Login.ngOnChanges" })
+  ngOnChanges(changes: SimpleChanges) {
+    // ...
+  }
 }
 ```
 
+Note: Due to code minification in production builds, you need to pass a `name` property to the `TraceMethod` decorator. Otherwise, created spans will refer to the component as `<unnamed>`
```

Comment:
```suggestion
Note: Due to code minification in production builds, you need to pass a `name` property to the `TraceMethod` decorator. Otherwise, created spans will refer to the component as `<unnamed>`.
```